### ISSUE TITLE
Create an airport selection form component

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#66]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [x] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

To select a destination airport in an event page template.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #65 

### Blockers

This issue is blocked by the following issue(s): #`issue number`

**Comment by fvolner** [comment]
[In repositories](https://github.com/Kentico/xperience-training-13/blob/acef058bad26663261d4b6aed3591cbc1787764b/Business/Repositories/AirportRepository.cs#L23) avoid sync over async if possible. Call a version of our API that retrieves data synchronously. If unavoidable replace `.GetAwaiter().GetResult()` with `.Result` and pray ;-)
